student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroomm = 'no classroom', **args)
+    super(**args)
+    @classroomm = classroomm
+  end
```

Comment:
- [ ] As the required for the project, the onstructor method of the `Student` class to extend parent's constructor by adding `@classroom` and a parameter for it. Kindly ensure that to update it in order to match the expected requirements. It should look similar to the following instead:
```ruby
class Student < Person 
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
  ...
```